### PR TITLE
[202012] Updating BRCM SAI to 4.3.5.1-9 for enabling SAI_SWITCH_ATTR_QOS_DSCP_TO_TC_MAP capability.

### PR DESCRIPTION
**Pull request by vmittal-msft**
#### Why I did it
To enable SAI attribute SAI_SWITCH_ATTR_QOS_DSCP_TO_TC_MAP capability.

#### How I did it
Made changes in SAI code to return TRUE for capability query by SONIC.

#### How to verify it
Tested the code on TD3 based box.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

#### Description for the changelog

Updating BRCM SAI to 4.3.5.1-9 for enabling SAI_SWITCH_ATTR_QOS_DSCP_TO_TC_MAP capability.

### CODE REVIEW

File: files/build/versions/default/versions-web [hunk 1]
```diff
@@ -28,6 +28,7 @@ https://github.com/barefootnetworks/sonic-release-pkgs/raw/dev/bfnsdk_20211112_s
 https://github.com/CentecNetworks/sonic-binaries/raw/master/amd64/libsai_1.6.3-1_amd64.deb==e7a41e5cf06b44ae3ed615d553de40d3
 https://github.com/CentecNetworks/sonic-binaries/raw/master/arm64/sai/libsai_1.6.3-1_arm64.deb==4121f09fbc5319cd488b832d72287f93
 https://github.com/CumulusNetworks/ifupdown2/archive/1.2.8-1.tar.gz==12f45e90d23178e96cf70f68dc9455e6
+https://github.com/CumulusNetworks/ifupdown2/archive/3.0.0-1.tar.gz==755459b3a58fbc11625336846cea7420
```
Comment (gechiang) [inline files/build/versions/default/versions-web:31]: this is strange... I am not expecting to see this change...

File: files/build/versions/default/versions-web [hunk 2]
```diff
@@ -36,6 +37,9 @@ https://github.com/nanomsg/nanomsg/archive/1.0.0.tar.gz==6f56ef28c93cee644e8c4aa
 https://github.com/NephosInc/SONiC/raw/master/sai/libsainps-dev_3.0.0_sai_1.5.0_06a67d_amd64.deb==64931041f82533e2a05e003ed9d878c8
 https://github.com/NephosInc/SONiC/raw/master/sai/libsainps_3.0.0_sai_1.5.0_06a67d_amd64.deb==385b84051259ebb2580522a990cca161
 https://github.com/NephosInc/SONiC/raw/master/sdk/npx_diag==a444c876aaf2a8b4c99238f15fbc13a0
+https://launchpad.net/debian/+archive/primary/+sourcefiles/bash/5.1-2/bash_5.1-2.debian.tar.xz==9d0cbd5f463f461c840c95f62a64d61b
```
Comment (gechiang) [inline files/build/versions/default/versions-web:40]: Nor these ones...

File: files/build/versions/default/versions-web [hunk 3]
```diff
@@ -50,13 +54,21 @@ https://sonicstorage.blob.core.windows.net/debian/pool/main/liby/libyang/libyang
 https://sonicstorage.blob.core.windows.net/debian/pool/main/n/net-snmp/net-snmp_5.7.3+dfsg-5.debian.tar.xz==ad957e90207d0669beb2109e4e325def
 https://sonicstorage.blob.core.windows.net/debian/pool/main/n/net-snmp/net-snmp_5.7.3+dfsg-5.dsc==2443e4dffbdb020e7ab4f947a7904912
 https://sonicstorage.blob.core.windows.net/debian/pool/main/n/net-snmp/net-snmp_5.7.3+dfsg.orig.tar.xz==6391ae27eb1ae34ff5530712bb1c4209
+https://sonicstorage.blob.core.windows.net/packages/20190307/bcmcmd?sv=2015-04-05&sr=b&sig=sUdbU7oVbh5exbXXHVL5TDFBTWDDBASHeJ8Cp0B0TIc%3D&se=2038-05-06T22%3A34%3A19Z&sp=r==b8aefc751bdf93218716bca6797460ff
```
Comment (gechiang) [inline files/build/versions/default/versions-web:57]: Nor this one...

File: files/build/versions/default/versions-web [hunk 4]
```diff
@@ -50,13 +54,21 @@ https://sonicstorage.blob.core.windows.net/debian/pool/main/liby/libyang/libyang
...
-https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.3/202012/libsaibcm_4.3.5.1-8_amd64.deb?sv=2020-04-08&st=2021-11-17T23%3A06%3A02Z&se=2035-11-18T23%3A06%3A00Z&sr=b&sp=r&sig=0%2BkErbtVjoER05lW5XrB%2B%2BdZ%2FBifcWgk9651zsYMKNw%3D==bd8b1d3cbd2ceaa2f7cb172cf3b2ea09
+https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.3/202012/libsaibcm-dev_4.3.5.1-9_amd64.deb?sv=2020-08-04&st=2021-12-07T19%3A32%3A45Z&se=2030-12-08T19%3A32%3A00Z&sr=b&sp=r&sig=sg2j65hKv%2BJsF%2BjATRlSxW415lT6Fa1cSaOlLIfZ7TM%3D==76bb2c1412a92ba66540e6bc2de7d653
+https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.3/202012/libsaibcm_4.3.5.1-9_amd64.deb?sv=2020-08-04&st=2021-12-07T19%3A29%3A13Z&se=2030-12-08T19%3A29%3A00Z&sr=b&sp=r&sig=0smiVHMjaLsVkr07u%2BpK9xcPFljhGKlnS92L2FGV0sc%3D==dd66f12c1a802fad58b625682456771e
+https://sonicstorage.blob.core.windows.net/packages/bcmsai/6.0/master/libsaibcm-dev_6.0.0.10_amd64.deb?sv=2020-08-04&st=2021-10-18T20%3A33%3A23Z&se=2030-10-19T20%3A33%3A00Z&sr=b&sp=r&sig=22zGsALAPStYDs42Ryegtf23zUvy9Hc0F9%2F6FOIdEn0%3D==a152ef80b7110aa5931715a81613aaec
```
Comment (gechiang) [inline files/build/versions/default/versions-web:61]: Nor this one and the rest of the changes in this file...

File: files/build/versions/default/versions-web (hunk 1 above)
Comment (vmittal-msft) [reply files/build/versions/default/versions-web:31]: @gechiang let me disucss with you offline.

**Review by gechiang — APPROVED** [review]

**Pull request merged**